roles/internal/crayfish/templates/gemini.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS gemini.Gemini (
```

Comment:
I'd like to see this use the variable for the db name as well, also based on the discussion above about idempotency it probably doesn't need the IF NOT EXISTS (although it doesn't hurt to have it there)
```sql
CREATE TABLE {{ gemini_db_options.dbname }}.Gemini (
    id INT UNSIGNED AUTO_INCREMENT PRIMARY KEY,
    drupal VARCHAR(2048) NOT NULL UNIQUE,
    fedora VARCHAR(2048) NOT NULL UNIQUE
) ENGINE=InnoDB;
```